packages/images/test/e2e/plugin-e2e-specs.js

```diff
@@ -10,11 +14,10 @@ chai.use(chaiAsPromised);
 chai.should();
 
 const THIS_PLUGIN_DIR = path.resolve(__dirname, '..', '..', '..');
-const APPIUM_HOME = path.resolve(__dirname, '..', '..', '..', 'local_appium_home');
 const TEST_HOST = 'localhost';
 const TEST_PORT = 4723;
-const TEST_FAKE_APP = path.resolve(APPIUM_HOME, 'appium-fake-driver', 'node_modules', 'appium-fake-driver', 'test', 'fixtures',
-                                   'app.xml');
+const TEST_FAKE_APP = path.resolve(APPIUM_HOME, '@appium', 'fake-driver', 'node_modules',
```

Comment:
this is brittle, but don't see a good alternative rn.  we need to be able to `require.resolve` anything...

Reply:
brittle because it relies on knowledge of npm's internal organization? yes.